src/test/scala/ai/privado/languageEngine/java/passes/config/PropertiesFilePassTest.scala

```diff
@@ -94,6 +94,57 @@ class AnnotationTests extends PropertiesFilePassTestBase(".properties") {
   }
 }
 
+/* Test for annotation of methods */
+class AnnotationMethodTests extends PropertiesFilePassTestBase(".yml") {
+  override val configFileContents: String =
+    """
+      |sample:
+      |  url: http://www.somedomain.com/
+      |""".stripMargin
+
+  override val propertyFileContents = ""
+  override val codeFileContents: String =
+    """
+      |
+      |import org.springframework.beans.factory.annotation.Value;
+      |
+      |class Foo {
+      |
+      |@Value("${sample.url}")
+      |public void setUrl( String sampleUrl )
+      |{
+      |    String url = sampleUrl;
+      |}
+      |}
+      |""".stripMargin
+
+  "ConfigFilePass" should {
+    "connect annotated method to property" in {
+      val anno: List[AstNode] = cpg.property.usedAt.l
+      anno.length shouldBe 1
+
+      anno.foreach(element => {
+        element.label match {
+          case "METHOD" =>
+            val List(methodNode: Method) = element.toList
+            methodNode.name shouldBe "setUrl"
+        }
+      })
+    }
+
+    "connect property to annotated method" in {
```

Comment:
I think this test can be simplified like below, easy to read as well.
```
cpg.property.usedAt.originalProperty.l.size shouldBe 1
cpg.property.usedAt.originalProperty.name.l shouldBe List("http://www.somedomain.com/")
```

Reply:
Okay, this was copied from the original test itself. I'll change it.